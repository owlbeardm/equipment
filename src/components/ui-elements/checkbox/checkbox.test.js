import React from 'react'
import { shallow } from 'enzyme'

import Checkbox from './checkbox'

test('renders without error', () => {
  const defaultProps = { input: { name: 'the-checkbox' }, label: 'the checkbox', disabled: false }
  const wrapper = shallow(<Checkbox {...defaultProps} />)
  expect(wrapper.html()).toMatchSnapshot()
})
