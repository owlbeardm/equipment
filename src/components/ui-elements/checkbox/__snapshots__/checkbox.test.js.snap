// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without error 1`] = `"<div class=\\"form-group\\"><div class=\\"form-check\\"><label class=\\"form-check-label text-dark\\"><input type=\\"checkbox\\" name=\\"the-checkbox\\" class=\\"form-check-input\\"/>the checkbox<span class=\\"form-check-sign\\"><span class=\\"check\\"></span></span></label></div></div>"`;
